content/chapters/compute/lecture/quiz/not-race-condition.md

```diff
@@ -0,0 +1,15 @@
+## Question Text
+
+What type of parallel actions never cause race conditions?
+
+## Question Answers
+
+- Only reads.
+- Reads and writes.
++ Reads and executions.
+- All parallel actions can cause race conditions
+
+## Feedback
+
+Race conditions only appear when 2 or more threads **write** data to the same address in parallel.
```

Comment:
No. Race conditions only appear when a thread writes and another thread depends on the write / update of the other thread.

A TOCTOU race condition is when a thread writes and another one reads **before** the update.